<launch>
    <node type="cluster_tracker.py" pkg="localization" name="cluster_tracking_node" output="screen">
        <!-- tolerance for cluster centroid movement in meters -->
        <param name="movement_tolerance" value="0.5" />
        
        <!-- amount of time [seconds] before impact predicted to alert operator -->
        <param name="impact_time_const" value="10" />

        <!-- minimum velocity to publish velocity vector to rviz -->
        <param name="min_velocity_threshold" value="0.3" />

    </node>

</launch>